### ISSUE TITLE
NVDA does not read comments in Microsoft Word with NVDA ALT C

### ISSUE THREAD

**Issue by math65** [nvaccess/nvda#11989]
### Steps to reproduce:

Create a comment in Microsoft Word version 365, and read with NVDA + ALT + C

### Actual behavior:

NVDA does not say anything

### Expected behavior:

Comment should be read

#### NVDA installed/portable/running from source:

Installed

#### NVDA version:

2020.3

#### Windows version:

Windows 10 version 20h2

#### Name and version of other software in use when reproducing the issue:

Microsoft Word version 365

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No

#### If addons are disabled, is your problem still occuring?

Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?

Yes

**Comment by zstanecic** [comment]
Could you please write down here the extact build number of the Microsoft word / office?

It can be found in the about dialog of the Microsoft word. 

Thanks,

Zvonimir

**Comment by math65** [comment]
Hi ! 

 

16.0.13530.20132

 

Mathieu MARTIN

 

 

De : zstanecic <notifications@github.com> 
Envoyé : vendredi 8 janvier 2021 09:47
À : nvaccess/nvda <nvda@noreply.github.com>
Cc : math65 <mathieumartin65@icloud.com>; Author <author@noreply.github.com>
Objet : Re: [nvaccess/nvda] NVDA does not read comments in Microsoft Word with NVDA ALT C (#11989)

 

Could you please write down here the extact build number of the Microsoft word / office?

It can be found in the about dialog of the Microsoft word. 

Thanks,

Zvonimir

**Comment by zstanecic** [comment]
I cannot reproduce this on Microsoft word 2019 /365 build (16.0.13530.20132), 64-bit version

Have you turned on Microsoft UIA support in the advanced settings panel, in the NVDA preferences dialog?

If so, please try to turn it off as a workaround for now…

**Comment by math65** [comment]
Yes, it’s work now !

 

Thank for your help !

 

De : zstanecic <notifications@github.com> 
Envoyé : vendredi 8 janvier 2021 10:16
À : nvaccess/nvda <nvda@noreply.github.com>
Cc : math65 <mathieumartin65@icloud.com>; Author <author@noreply.github.com>
Objet : Re: [nvaccess/nvda] NVDA does not read comments in Microsoft Word with NVDA ALT C (#11989)

 

I cannot reproduce this on Microsoft word 2019 /365 build (16.0.13530.20132), 64-bit version

Have you turned on Microsoft UIA support in the advanced settings panel, in the NVDA preferences dialog?

If so, please try to turn it off as a workaround for now…

**Comment by zstanecic** [comment]
Ok. Feel free to close this.
